WordPress/gutenberg
# update empty patterns

https://github.com/WordPress/gutenberg/assets/1072756/3b8c83b0-5fed-4201-aee4-73ea4ff4843a



## What?
Updates the empty state when no patterns exist.
## Why?
Direct people to create their first pattern.

https://github.com/WordPress/gutenberg/issues/53287

## How?
Updates the no-patterns component and adds a renderTrigger prop to the AddNewPattern component.

## Testing Instructions
1. Go to site editor
2. Open patterns page when no patterns are available
3. Notice empty state in content area on right
4. Click "new pattern" and notice pattern creation modal
